docs/geedocs/docsrc/answer/developers/creatingFileAccessorPlugin.rst

```diff
@@ -0,0 +1,56 @@
+|Google logo|
```

Comment:
We should state in here somewhere that this is experimental and not guaranteed to work correctly.

Reply:
Good idea. I put a note about that at the top of the doc.